make/addons.mk

```diff
@@ -25,7 +25,7 @@ export KUBE_VIP_VERSION := v0.8.3
 
 export METALLB_CHART_VERSION := 0.14.8
 
-export COSI_CONTROLLER_VERSION := 0.0.1-alpha.4
+export COSI_CONTROLLER_VERSION := 0.0.1-alpha.5
```

Comment:
This version doesn't defaults to the necessary priority class, etc? No need for overrides here?

Reply:
No need for overrides; the chart has the defaults we need: https://github.com/mesosphere/charts/blob/2970620efaa1ad449abe0025ba803a3b26d70d47/stable/cosi/values.yaml#L10-L18